app/views/documents/_attachment.html.erb

```diff
@@ -70,11 +71,11 @@
     <% end %>
 
     <% unless attachment.accessible? %>
-      <div class="accessibility-warning js-toggle-accessibility-warning" id="<%= help_block_id %>">
+      <div data-module="toggle" class="accessibility-warning" id="<%= help_block_id %>">
         <h2><%= t('attachment.accessibility.heading') %>
-          <span class="toggler"><%= t('attachment.accessibility.request_a_different_format') %></span>
+          <a class="toggler" href="#<%= help_block_toggle_id %>" data-controls="<%= help_block_toggle_id %>" data-expanded="false"><%= t('attachment.accessibility.request_a_different_format') %></a>
         </h2>
-        <p class="help-block">
+        <p id="<%= help_block_toggle_id %>" class="hidden">
```

Comment:
Yeah, it was hidden by the WH toggler JS, so this would be a regression for non-JS.

I think this would need to be conditional on a `.js-hidden` type class, or hiding it becomes the responsibility of the toggler module, what do you think?